ui/StatusQ/src/StatusQ/Core/Utils/Utils.qml

```diff
@@ -267,4 +267,24 @@ QtObject {
     function stripHttpsAndwwwFromUrl(text) {
         return text.replace(/http(s)?(:)?(\/\/)?|(\/\/)?(www\.)?(\/)/gim, '')
     }
+
+    function getSelectedTextWithFormationChars(messageInputField) {
```

Comment:
Doesn't have to be here at all imho, afaics it's only used inside StatusChatInput (and its test)

Reply:
This function itself is really weird reading it I'm pretty sure it's buggy. But it's rather different story to go deeper into that thing. It could be placed `StatusChatInput` indeed, it's not used outside (it's also not sued in tests). If you want I can move, but imo it doesn't hurt when stays here. Wdyt?

Reply:
Yeah it doesn't matter in the end, just thought we should rather reduce the size of this "dump all" Utils.qml :)

Reply:
Yeah, we should. It's not a big thing but you are right. I moved to `StatusChatInput`.